parser.cs

```diff
@@ -150,14 +150,14 @@ public class Parser {
 
     private void type(){
         switch(__lookahead.Type){
-            case TOKENS.INTEGER:
-                match(TOKENS.INTEGER);
```

Comment:
On second thought, these should be INTEGER, FLOAT, STRING, etc because of rules 10-13. INTEGER_LIT is 023424, INTEGER is "Integer"